chapters/extensions/cleanup.md

```diff
@@ -0,0 +1,118 @@
+
+# VK_EXT_host_query_reset
+
+> Promoted to core in Vulkan 1.2
+
+This extension allows an application to call `vkResetQueryPool` from the host instead of needing to setup logic to submit `vkCmdResetQueryPool` since this is mainly just a quick write to memory for most implementations and little overhead to calling.
+
+# VK_KHR_separate_depth_stencil_layouts
+
+> Promoted to core in Vulkan 1.2
+
+This extension allows an application when using a depth/stencil format to do an image translation on each the depth and stencil separately. Starting in Vulkan 1.2 this functionality is required for all implementations.
+
+# VK_EXT_separate_stencil_usage
+
+> Promoted to core in Vulkan 1.2
+
+There are formats that express both the usage of depth and stencil, but there was no way to list a different usage for them. The `VkImageStencilUsageCreateInfo` now lets an application pass in a separate `VkImageUsageFlags` for the stencil usage of an image. The depth usage is the original usage passed into `VkImageCreateInfo::usage` and without using `VkImageStencilUsageCreateInfo` the stencil usage will be the same as well.
+
+A good use case of this is when using the [VK_KHR_image_format_list](./VK_KHR_image_format_list.md) extension. This provides a way for the application to more explicitly describe the possible image views of their `VkImage` at creation time. This allows some implementations to possibly do implementation dependent optimization depending on the usages set.
+
+# VK_KHR_dedicated_allocation
+
+> Promoted to core in Vulkan 1.1
+
+Normally applications allocate large chunks for `VkDeviceMemory` and then suballocate to various buffers and images. There are times where it might be better to have a dedicated allocation for `VkImage` or `VkBuffer`. An application can pass `VkMemoryDedicatedRequirements` into `vkGetBufferMemoryRequirements2` or `vkGetImageMemoryRequirements2` to fine out if a dedicated allocation is preferred or required. When dealing with external memory it will often require a dedicated allocation.
+
+# VK_EXT_sampler_filter_minmax
+
+> Promoted to core in Vulkan 1.2
+
+By default in Vulkan sampler filters to an image lookup to return a filtered texel value produced by computing a weighted average of a collection of texels in the neighborhood of the texture coordinate provided. This extension provides a new sampler parameter which allows applications to produce a filtered texel value by computing a component-wise minimum (`VK_SAMPLER_REDUCTION_MODE_MIN`) or maximum (`VK_SAMPLER_REDUCTION_MODE_MAX`) of the texels that would normally be averaged. This is similar to [GL EXT_texture_filter_minmax](https://www.khronos.org/registry/OpenGL/extensions/EXT/EXT_texture_filter_minmax.txt).
```

Comment:
I can't quite parse the first sentence. Maybe:
"By default, Vulkan samplers using linear filtering return a filtered texel value produced by computing a weighted average of a collection of texels in the neighborhood of the texture coordinate provided."